Makefile

```diff
@@ -24,9 +21,8 @@ install_python_deps: ## Install all python dependencies/requirements
 
 .PHONY: ensure_pre_commit
 ensure_pre_commit: .git/hooks/pre-commit  ## Ensure pre-commit is installed
-.git/hooks/pre-commit: $(PRE_COMMIT)
-	pre-commit install
-	pre-commit install-hooks
+.git/hooks/pre-commit: $(shell which pre-commit)
```

Comment:
Changed to using the shell to get the path because it's easier to maintain. Otherwise it would have changed again to account for installing via `pip` instead of `brew`

Reply:
It looks like `make setup` calls `pipenv install`, which installs `pre-commit`.  Do we still need this target then?

Reply:
`pipenv` installs the pre-commit package, but this command installs the hooks that pre-commit will run.